custom/conf/app.ini.sample

```diff
@@ -74,6 +74,34 @@ WORK_IN_PROGRESS_PREFIXES=WIP:,[WIP]
 ; List of reasons why a Pull Request or Issue can be locked
 LOCK_REASONS=Too heated,Off-topic,Resolved,Spam
 
+[repository.signing]
+; GPG key to use to sign commits, Defaults to the default - that is the value of git config --get user.signingkey
+; Switch to none to stop signing completely
+SIGNING_KEY = default
+; If a SIGNING_KEY ID is provided, use the provided Name and Email address as the signer.
+; These should match a publicized name and email address for the key
+SIGNING_NAME =
+SIGNING_EMAIL =
```

Comment:
Can these be derived from git config as well, as a default?

Reply:
When the SIGNING_KEY is set to default these values are not used and instead the results of: `git config --get user.name` and `git config --get user.email` are used.

I've adjusted the comments here.